telegram/forcereply.py

```diff
@@ -22,17 +22,25 @@
 
 
 class ForceReply(ReplyMarkup):
-    """This object represents a Telegram ForceReply.
+    """
+    Upon receiving a message with this object, Telegram clients will display a reply interface to
+    the user (act as if the user has selected the bot's message and tapped 'Reply'). This can be
+    extremely useful if you want to create user-friendly step-by-step interfaces without having
+    to sacrifice privacy mode.
```

Comment:
In this case it is pretty self-explanatory. Users can look into tg's docs on how to build reply_markups. They have a page for that. If you want more explanatory stuff I think it should go in the wiki. This describes the object ForceReply.